src/app/static/src/app/components/modelOptions/ModelOptions.vue

```diff
@@ -1,28 +1,34 @@
 <template>
-    <dynamic-form :form-meta="form"
+    <dynamic-form v-model="modelOptions"
                   submit-text="Validate"
-                  @submit="validated"></dynamic-form>
+                  @submit="validate"></dynamic-form>
 </template>
 <script lang="ts">
     import Vue from "vue";
     import DynamicForm from "../forms/DynamicForm.vue";
-    import {DynamicFormMeta, formMeta} from "../forms/types";
-    import {mapActionsByNames} from "../../utils";
-
+    import {DynamicFormData, DynamicFormMeta} from "../forms/types";
+    import {mapMutationsByNames} from "../../utils";
     interface Methods {
-        validated: () => void
+        validate: (data: DynamicFormData) => void
+        update: (data: DynamicFormMeta) => void
     }
-
-    export default Vue.extend<{ form: DynamicFormMeta }, { validated: () => void }, {}, {}>({
+    export default Vue.extend<{ form: DynamicFormMeta }, Methods, {modelOptions: DynamicFormMeta}, {}>({
         name: "ModelOptions",
-        data() {
-            return {form: formMeta}
+        computed: {
+            modelOptions: {
+                get () {
+                    return this.$store.state.modelOptions.optionsFormMeta
+                },
+                set (value: DynamicFormMeta) {
+                    this.update(value);
+                }
+            }
         },
         methods: {
-            ...mapActionsByNames<keyof Methods>("modelOptions", ["validated"])
+            ...mapMutationsByNames<keyof Methods>("modelOptions", ["validate", "update"])
```

Comment:
Mapping the mutations directly here, but once we're validating from the server will want to introduce a "validate" action.